hw/hw_04_task_04.py

```diff
@@ -0,0 +1,45 @@
+from typing import List
+
+
+# flake8: noqa: CCR001
+def fizzbuzz(n: int) -> List[str]:
+    """
+    Function takes a number N as an input and returns N FizzBuzz numbers.
+
+    Instruction:
+     - Install Python 3.8 (https://www.python.org/downloads/)
+     - Install pytest `pip install pytest`
```

Comment:
Why do I need pytes?

Reply:
I don't, my bad.

Reply:
Why `pytest` was in the instruction template?